app/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("api/cinema/", include("cinema.urls", namespace="cinema")),

```

Comment:
The `include` function with a `namespace` argument requires that the included URLconf defines an `app_name` attribute. Ensure that the `cinema.urls` module has an `app_name` attribute defined to avoid runtime errors.